app/models/setting.rb

```diff
@@ -269,10 +275,19 @@ def self.cache
 
   # Methods for loading default settings
 
+  def self.default_settings
+    []
+  end
```

Comment:
One note - this method is new, all plugins which will start using this method must bump the minimum required Foreman version in the `engine.rb` file to 1.24. Can you update all your PRs please?

Reply:
Oh snap.... I don't want to, but I'll :frowning_man: